docker/start-docker.sh

```diff
@@ -1,50 +1,51 @@
-export DOCKER_IMAGE=intelanalytics/bigdl-ppml-trusted-fastchat-tdx:test
-
-export MODEL_PATH="YOUR_MODEL_PATH"
-
+export MODEL_PATH=YOUR_MODE_PATH
+export DOCKER_IMAGE=10.239.45.10/arda/intelanalytics/bigdl-llm-serving-cpu:2.4.0-SNAPSHOT
 sudo docker run -itd \
         --net=host \
-        --cpuset-cpus="0-3" \
+        --cpuset-cpus="Normally 4 cores" \
         --cpuset-mems="0" \
         --memory="16G" \
         --name=fastchat-controller \
         --shm-size="16g" \
-        -e CONTROLLER_HOST="172.168.0.218" \
+        -e CONTROLLER_HOST="YOUR_HOST_IP" \ # Change this
         -e CONTROLLER_PORT="21005" \
-        -e API_HOST="172.168.0.218" \
+        -e API_HOST="YOUR_HOST_IP" \ # Change this
         -e API_PORT="8000" \
+        -e ENABLE_PERF_OUTPUT=true \
         -v $MODEL_PATH:/ppml/models \
         $DOCKER_IMAGE -m controller
 
-
 sudo docker run -itd \
         --net=host \
-        --cpuset-cpus="5-20" \
+        --cpuset-cpus="Normally 48 cores" \
         --cpuset-mems="0" \
-        --memory="128G" \
+        --memory="64G" \
         --name=fastchat-worker-1 \
         --shm-size="16g" \
-        -e CONTROLLER_HOST="172.168.0.218" \
+        -e CONTROLLER_HOST="YOUR_HOST_IP" \ # Change this
         -e CONTROLLER_PORT="21005" \
-        -e WORKER_HOST="172.168.0.218" \
+        -e WORKER_HOST="YOUR_HOST_IP" \ # Change this
         -e WORKER_PORT="21840" \
-        -e MODEL_PATH="/ppml/models/llama-7b-bigdl/" \
```

Comment:
Why we delete this `MODEL_PATH` environment variable

Reply:
Sorry ,it's a mistake.